### DISCUSSION TITLE
Trying to load dags dynamically from helper modules: DAGS not showing up on airflow UI

### DISCUSSION THREAD

**Discussion by spoorti-nidagundi** [apache/airflow#26286] [Q&A]
How do I ensure proper loading of modules occurs? 
I have set the path too and it's in the allowed list of directoreis that airflow allows. Needed help on this

**Comment by potiuk** [comment]
Study those thoroughly: 
* https://airflow.apache.org/docs/apache-airflow/stable/modules_management.html 
* https://airflow.apache.org/docs/apache-airflow/stable/howto/dynamic-dag-generation.html?highlight=dynamic+dags

Generally, our documentation is pretty helpful. And if you need help., I suggest that you follow  https://dontasktoask.com/ .

Also read it thoroughly to understand why asking for someone to help you without specifying what problem you have in detail is a bad idea.
